### PR TITLE
[ViewJob]: Add ViewJob component

### PR DESCRIPTION
**Pull request by koushikmohan1996**
# Pull Request Template

## Description
Added view job component

Fixes #9 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by koushikmohan1996** [comment]
viewjob

File: frontend/src/Containers/ViewJob/Notes/AddNote.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { Dialog, DialogContent, DialogTitle } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { Button, Input } from '../../../Material-UI/Components';
+import APIService from '../../../service/APIService';
+
+const useStyles = makeStyles(({
+  error: {
+    color: 'rgb(201, 58, 58)',
```
Comment (hiteshkundal) [inline frontend/src/Containers/ViewJob/Notes/AddNote.js]: use theme color for error as 
```
const useStyles = makeStyles((theme) => ({
error : {
color : theme.palette.error.main,
}
}))
```

File: frontend/src/Containers/ViewJob/Notes/Notes.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  },
+  notes: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  card: {
+    width: 250,
+    height: 250,
+    margin: 5,
+    border: '1px solid #0000001a',
```
Comment (hiteshkundal) [inline frontend/src/Containers/ViewJob/Notes/Notes.js]: Add this color to theme color and then use

File: frontend/src/Containers/ViewJob/Notes/Notes.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  addBox: {
+    display: 'flex',
+    alignItems: 'center',
+    alignContent: 'center',
+    justifyContent: 'center',
+    width: 250,
+    height: 250,
+    margin: 5,
+    cursor: 'pointer',
+    border: '1px solid #0000001a',
```
Comment (hiteshkundal) [inline frontend/src/Containers/ViewJob/Notes/Notes.js]: same as above

**Pull request merged**